theano/gpuarray/opt.py

```diff
@@ -282,15 +282,17 @@ def apply(self, fgraph):
                 continue
 
             # Move only if any of the inputs are on the GPU.
-            move_to_GPU = False
+            move_to_GPU = True
 
+            '''
             if any([isinstance(i, GpuArrayVariable) or
                     isinstance(i, GpuArraySharedVariable)
                     for i in [mapping[v] for v in node.inputs] +
                               node.outputs]):
 
                 move_to_GPU = True
-
```

Comment:
Ignore these changes for now. I made it for profiling without the initial condition